app/utils/press-release-snapshot.js

```diff
@@ -53,6 +56,17 @@ export default class PressReleaseSnapshot {
     this.pressRelease.governmentFields = this.governmentFields;
     this.pressRelease.themes = this.themes;
     this.pressRelease.sources = this.sources;
+
+    //destroys and record which is added but not saved
+    if (this.pressRelease.attachments.length) {
+      this.pressRelease.attachments.forEach(attachment => {
+        const isNewlyAdded = this.attachments.indexOf(attachment) === -1;
+        if (isNewlyAdded) {
+          attachment.destroyRecord();
```

Comment:
`await` toevoegen. Als je hier dan met `.map()` werkt ipv `forEach` kan je de promises in parallel laten lopen en awaiten met `Promise.all`